tajo-catalog/tajo-catalog-common/src/main/proto/CatalogProtos.proto

```diff
@@ -181,6 +181,7 @@ message TableStatsProto {
   optional int64 readBytes = 7;
   repeated ColumnStatsProto colStat = 8;
   optional int32 tid = 9;
+  repeated PartitionDescProto partitions = 10;
```

Comment:
Could you keep PartitionDescProto in TableStatsProto? Other messages like TaskCompletionReport may be a better place.